index.html

```diff
@@ -66,35 +66,35 @@ <h2 class="lead secondary-heading">
     <div class="row">
       <!-- 1 -->
       <div class="col-lg-4 col-md-6 feature">
-        <h3>Julia is fast!</h3>
+        <h3>Fast</h3>
         <p>Julia was designed from the beginning for <a href="/benchmarks/">high performance</a>.
         Julia programs compile to efficient native code for <a href="/downloads/#support_tiers">multiple platforms</a> via LLVM.</p>
       </div>
       <!-- 2 -->
       <div class="col-lg-4 col-md-6 feature">
         <h3>Dynamic</h3>
-        <p>Julia is <a href="https://docs.julialang.org/en/v1/manual/types/">dynamically typed</a>, feels like a scripting language, and has good support for <a href="https://docs.julialang.org/en/v1/stdlib/REPL/">interactive</a> use.</p>
+        <p>Julia is <a href="https://docs.julialang.org/en/v1/manual/types/">dynamically typed</a>, feels like a scripting language, and has good support for <a href="https://docs.julialang.org/en/v1/stdlib/REPL/">interactive</a> use. Julia has a <a href="https://docs.julialang.org/en/v1/manual/types/">rich language of descriptive datatypes</a>, and type declarations can be used to clarify and solidify programs.</p>
```

Comment:
It doesn't really make sense to mention type declarations under "dynamic". I guess it could go under "fast" or "general".

Reply:
Possible solution: rename section to "Dynamic and optionally typed"?

Reply:
That would be too long a name for the page design.

Reply:
I am going to leave out the types and optional typing. That seems like too much detail at this stage of someone's Julia journey.